### PR TITLE
[Client] 모바일 반응형

### PR DESCRIPTION
**Pull request by yoosion030**
## 개요 💡

> 모바일 반응형 스타일을 추가했습니다.

## 작업내용 ⌨️

1. 메인 페이지 반응형 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/ded123ebefccddebd98e0e55e55a711a63fa77a9, https://github.com/themoment-team/official-gsm-front/pull/176/commits/efa7da56bc85acf898e7de43b7480348020d3fd2
2. 헤더 반응형 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/839f17e906183385fcdd329c3d4f97d4644b093c
3. 아이콘 단위 수정 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/aabaf402b1abc2a38db178473f89527f9bd9f3f7
4. 리스트 페이지 반응형 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/9a863ac6405216810f031a180270b2d826f178dc, https://github.com/themoment-team/official-gsm-front/pull/176/commits/920a0f7a0f52d057929d5941859694525f052f9f
5. 학교소개 section2, 5 반응형 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/a85bcd52e9ff9a60a4e5b74d4f892af824fd6af4, https://github.com/themoment-team/official-gsm-front/pull/176/commits/2dd7dc400f0ac002b5122137eab458d051d92f25
6. root 폰트사이즈 수정 - https://github.com/themoment-team/official-gsm-front/pull/176/commits/9230022a283e4e105e5af73579203c2579a0030d

스크린샷 2023-07-16 오후 11 48 29
스크린샷 2023-07-16 오후 11 48 24
스크린샷 2023-07-16 오후 11 48 17
스크린샷 2023-07-16 오후 11 48 05

### CODE REVIEW

**Review by handsomejeongwoo — APPROVED** [review]

**Review by Lee-Seungje — APPROVED** [review]

**Review by yebin0310 — APPROVED** [review]

File: apps/client/src/components/CategoryHeader/style.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const CategoryHeaderWrapper = styled.div`
   height: 2.5625rem;
   display: flex;
   justify-content: space-between;
-  align-items: end;
+  align-items: center;
```
Comment (hyeongrok7874) [inline apps/client/src/components/CategoryHeader/style.ts:10]: 스크린샷 2023-07-19 오후 5 41 01

디자인대로라면 기존 코드 그대로 두는게 맞다고 생각됩니다!

**Review by hyeongrok7874 — APPROVED** [review]
수고하셨습니다!

File: apps/client/src/components/CategoryHeader/style.ts (hunk 1 above)
Comment (yoosion030) [reply apps/client/src/components/CategoryHeader/style.ts:10]: 아 그렇군요 디자인을 확인 못했습니다.
수정할게요

Comment (yoosion030) [reply apps/client/src/components/CategoryHeader/style.ts]: https://github.com/themoment-team/official-gsm-front/pull/176/commits/be1b71751e6fd03ba3862a61aaef98c5f5ebec18
수정했습니다!

**Pull request merged**